core/designsystem/src/main/java/com/hilingual/core/designsystem/component/button/HilingualButton.kt

```diff
@@ -0,0 +1,99 @@
+package com.hilingual.core.designsystem.component.button
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.common.extension.noRippleClickable
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import kotlinx.coroutines.delay
+
+@Composable
+fun HilingualButton(
+    text: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true
+) {
+    val backgroundColor = if (!enabled) {
+        HilingualTheme.colors.gray300
+    } else {
+        HilingualTheme.colors.black
+    }
+    val textColor = HilingualTheme.colors.white
+    Box(
+        modifier = modifier
+            .width(328.dp)
+            .height(58.dp)
+            .background(
+                color = backgroundColor,
```

Comment:
p2
```suggestion
                color = if (!enabled) HilingualTheme.colors.gray300 else  HilingualTheme.colors.black,
```

추가로 조건문에는 긍정을 기본으로 하는게 가독성에 좋다고 생각합니다ㅎㅎ if(true)

Reply:
엇 긍정을 기본으로 하려면 `color = if (enabled) HilingualTheme.colors.black else  HilingualTheme.colors.black.gray300` 이렇게 바꾸라는 말씀이실까요 ?!?! 처음 슬롯에 enabled = true로 설정해놔서요 !!

Reply:
네네 코드는 말그대로 저런 방법도 있다~정도고 기존 코드는 부정을 조건으로 쓰고있었는데 긍정을 기본으로 하면 좋다는 뜻!